validator/rpc/handlers_health.go

```diff
@@ -0,0 +1,163 @@
+package rpc
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	http2 "github.com/prysmaticlabs/prysm/v4/network/http"
+	pb "github.com/prysmaticlabs/prysm/v4/proto/prysm/v1alpha1"
+	"github.com/prysmaticlabs/prysm/v4/runtime/version"
+	"go.opencensus.io/trace"
+	"google.golang.org/protobuf/types/known/emptypb"
+)
+
+// GetVersion returns the beacon node and validator client versions
+func (s *Server) GetVersion(w http.ResponseWriter, r *http.Request) {
+	ctx, span := trace.StartSpan(r.Context(), "validator.web.GetVersion")
+	defer span.End()
+
+	beacon, err := s.beaconNodeClient.GetVersion(ctx, &emptypb.Empty{})
+	if err != nil {
+		http2.HandleError(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+
+	http2.WriteJson(w, struct {
+		Beacon    string `json:"beacon"`
+		Validator string `json:"validator"`
+	}{
+		Beacon:    beacon.Version,
+		Validator: version.Version(),
+	})
+}
+
+// StreamBeaconLogs from the beacon node via a gRPC server-side stream.
```

Comment:
```suggestion
// StreamBeaconLogs from the beacon node via server-side events.
```
Is this more accurate?